Tests/Helper/NumberHelperTest.php

```diff
@@ -97,6 +97,12 @@ public function testArguments()
      */
     public function testExceptionOnInvalidParams()
     {
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
So I don't understand why they are using this.

IMO, [`version_compare`](http://php.net/manual/fr/function.version-compare.php) is much easier to read and understand.

cc @rande what is the convention in sonata for that?